subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -3,11 +3,7 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-menuconfig NRF_CLOUD
-	bool "nRF Cloud Library"
-	select CJSON_LIB
-
-if NRF_CLOUD
+if NRF_CLOUD_MQTT || NRF_CLOUD_REST || NRF_CLOUD_PGPS
```

Comment:
what is the case where `NRF_CLOUD_PGPS` needs the host name or sec tag, but MQTT or REST is not already enabled?

Reply:
When the application is configured for either AWS IoT or Azure IoT Hub. In those cases we are not dependent on the nRF Cloud MQTT transport to receive/request A/P-GPS data. I included NRF_CLOUD_PGPS to avoid build error when including the nRF Cloud PGPS library.